test/src/main/webapp/META-INF/context.xml

```diff
@@ -12,7 +12,7 @@
 		name="jdbc/ibis4test-oracle"
 		type="javax.sql.DataSource"
 		driverClassName="oracle.jdbc.driver.OracleDriver"
-		url="jdbc:oracle:thin:@localhost:1521:xe"
+		url="jdbc:oracle:thin:@${databaseHost}:1521:xe"
```

Comment:
Als je databaseHost ergens een waarde geeft (dat zie ik nergens gebeuren, overigens), dan kan je daar ``host.docker.internal`` voor gebruiken.

Reply:
In test/pom.xml staat:

    &lt;databaseHost&gt;localhost&lt;/databaseHost&gt;

Ik denk dat dit in een eerdere commit is gebeurd die al gemerged is. Deze regel moet denk ik zo blijven, omdat je anders niet meer met Eclipse het FF! kan starten. In dat geval moet er `localhost` in de database URL staan.

Reply:
Ik heb Gerrit gisteren gesproken over alle feedback items. Ik heb gehoord dat we geen handmatige acties willen vragen zoals ik gedaan heb door een README.md te schrijven. Gerrit heeft gezegd dat Maven dit werk moet doen, maar Niels heeft ooit gezegd dat we geen Docker containers willen starten vanuit Maven. Niels en Gerrit, kunnen jullie beslissen hoe het moet worden?